### PR TITLE
[pypi]: Fix broken badges

### LINKED ISSUE #2022: My PyPI Project: "invalid json response"
[PyPI](https://pypi.org/project/hookmeup/)

I'm not a JS expert, but I can't find anything wrong with either the JSON PyPI is presenting or the parsing code.

### PR DESCRIPTION
**Pull request by paulmelnikow**
- When license is empty, get license from trove classifiers
- Allow empty classifiers array

Closes #2022

### CODE REVIEW

File: services/pypi/pypi-helpers.js [hunk 1]
```diff
@@ -50,6 +50,27 @@ const parseClassifiers = function(parsedData, pattern) {
...
 }
 
+function getLicenses(packageData) {
+  const {
+    info: { license },
+  } = packageData
+  if (license) {
+    return [license]
+  } else {
+    const acronymRegex = /\([^)+]\)/
```
Comment (espadrine) [inline services/pypi/pypi-helpers.js]: Don't we want the `+` to be after the `[`?

File: services/pypi/pypi-helpers.js [hunk 2]
```diff
@@ -50,6 +50,27 @@ const parseClassifiers = function(parsedData, pattern) {
...
+    let licenses = parseClassifiers(packageData, /^License :: (.+)$/)
+      .map(classifier => classifier.split(' :: ').pop())
+      .map(license => {
+        const match = license.match(acronymRegex)
+        return match ? match[1] : license
+      })
+    if (licenses.length > 1) {
+      licenses = licenses.filter(l => l !== 'dfsg approved')
```
Comment (espadrine) [inline services/pypi/pypi-helpers.js]: Why do we exclude DFSG?

Comment (espadrine) [reply services/pypi/pypi-helpers.js]: Ah, got it, they don't specify the exact license there.

File: services/pypi/pypi-helpers.spec.js [hunk 3]
```diff
@@ -100,6 +101,29 @@ describe('PyPI helpers', function() {
...
+      given({
+        info: {
+          license: '',
+          classifiers: [
+            'License :: OSI Approved :: MIT License',
+            'License :: DFSG approved',
+          ],
+        },
+      }),
+    ]).expect(['mit license'])
```
Comment (espadrine) [inline services/pypi/pypi-helpers.spec.js]: Could we have an example for the acronym case? I assume this is for eg. [magma](https://pypi.org/pypi/magma/json)?

File: services/pypi/pypi-helpers.js (hunk 1 above)
Comment (paulmelnikow) [reply services/pypi/pypi-helpers.js]: Ha, yea, that is totally broken. We don't want the parentheses, either.

**Comment by paulmelnikow** [comment]
Updated!

**Comment by paulmelnikow** [comment]
Thanks!

**Pull request merged**